src/quo2/components/drawers/bottom_actions/style.cljs

```diff
@@ -0,0 +1,27 @@
+(ns quo2.components.drawers.bottom-actions.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(def buttons-container
+  {:flex-direction     :row
+   :justify-content    :space-around
+   :padding-horizontal 20
+   :padding-vertical   12})
+
+(def button-container
+  {:flex 1})
+
+(def button-container-2-actions
```

Comment:
Maybe `button-container-two-actions`

Reply:
I wanted to keep component open to a possibility of changing the buttons amount, so added numeric count to navigate through bigger number of button names easier